Functions/Example/Tests/FUNSerializerTests.m

```diff
@@ -82,7 +82,12 @@ - (void)testDecodeInvalidLong {
   FUNSerializer *serializer = [[FUNSerializer alloc] init];
   NSDictionary *input = @{
     @"@type" : @"type.googleapis.com/google.protobuf.Int64Value",
+#if TARGET_OS_MACCATALYST
+    // NSNumberFormatter is more robust? on Catalyst
+    @"value" : @"some junk",
```

Comment:
I don't think this is the right way to fix this. The original value of this was invalid and should fail, even if that means not using NSNumberFormatter as-is. Let me think on it some more.

Reply:
Maybe a bug in the beta related to  https://developer.apple.com/documentation/foundation/nsnumberformatter/1415137-defaultformatterbehavior?language=objc? I tried setting to https://developer.apple.com/documentation/foundation/nsnumberformatterbehavior/nsnumberformatterbehavior10_4?language=objc, but no difference.

Reply:
Just wanted to check in and see if this still happens on the public version of Catalina and what the right approach moving forward should be (if `NSNumberFormatter` is in fact more robust through Catalyst.

Reply:
So running this test on catalyst in Xcode 12 (and Xcode 13) with no modifications passes. Im thinking this means that this was likely a bug in the beta this issues surfaced in and is no longer an issue– WDYT? @ryanwilson

Reply:
Do you mean you ran the tests as is (without this PR) and everything was fine?

Reply:
Yep, exactly.

Reply:
Wicked! :shipit: thanks for checking